core/src/main/scala/chisel3/internal/firrtl/IR.scala

```diff
@@ -382,7 +382,7 @@ abstract class Component extends Arg {
 @nowarn("msg=class Port") // delete when Port becomes private
 @deprecated(deprecatedPublicAPIMsg, "Chisel 3.6")
 case class DefModule(id: RawModule, name: String, ports: Seq[Port], commands: Seq[Command]) extends Component {
-  val secretConnects: mutable.ArrayBuffer[Connect] = mutable.ArrayBuffer[Connect]()
+  val secretConnects: mutable.ArrayBuffer[Command] = mutable.ArrayBuffer[Command]()
```

Comment:
If this is now allowing for arbitrary Chisel IR to be added, doing the name change to `secretIR` or `secretCommands` would be better.